js/piemenus.js

```diff
@@ -3658,14 +3658,27 @@ const piemenuGrid = (activity) => {
     activity.turtles._exitWheel.createWheel(["×", " "]);
 
     activity.turtles._exitWheel.navItems[0].navigateFunction = () => {
-        docById("wheelDivptm").style.display = "none";
-        activity.turtles.gridWheel.removeWheel();
-        activity.turtles._exitWheel.removeWheel();
+        hidePiemenu(activity);
     };
 
     if (docById("helpfulWheelDiv").style.display !== "none") {
         docById("helpfulWheelDiv").style.display = "none";
     }
+
+    const hidePiemenu = (activity) => {
+        docById("wheelDivptm").style.display = "none";
+        activity.turtles.gridWheel.removeWheel();
+        activity.turtles._exitWheel.removeWheel();
+    };
+     const clickOutsideHandler = (event) => {
```

Comment:
Please add a new line between these definitions (hidePiemenu and clickOutsideHandler) and please remove the extra space of indentation.

Reply:
Sir I did the change ..Is it okay or something else to do